packages/strapi-plugin-documentation/controllers/Documentation.js

```diff
@@ -18,19 +18,14 @@ const koaStatic = require('koa-static');
 module.exports = {
   getInfos: async ctx => {
     try {
-      const prefix = _.get(
-        strapi.plugins,
-        ['documentation', 'config', 'x-strapi-config', 'path'],
-        '/documentation'
-      );
       const service = strapi.plugins.documentation.services.documentation;
       const docVersions = service.retrieveDocumentationVersions();
       const form = await service.retrieveFrontForm();
 
       ctx.send({
         docVersions,
         currentVersion: service.getDocumentationVersion(),
-        prefix: `/${prefix}`.replace('//', '/'),
+        prefix: strapi.plugins.documentation.config['x-strapi-config'].path,
```

Comment:
Yes it is in purpose because in many other places the code uses `strapi.plugins.documentation.config['x-strapi-config'].path` directly, so it would be already broken and since it's by default "/documentation", it should never be not defined (except if the user overwrites it on purpose).

Also I would prefer that the default value is defined at a unique place to avoid rewritting '/documentation' each time and to avoid changing in at one place but not in the others.
What do you think?

Reply:
You surely can setup some defaults in the bootstrap of the plugin for sure :) This would be great I think.

Reply:
As we discussed it will be handle later in the new config system